Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,67 @@
+#include <iostream>
+#include <string>
+#include "Person.h"
+using namespace std;
+
+
+
+Person::Person(){
+    age_ = new int(1);
+}
+
+Person::~Person()
+{
+    delete age_;
+}
+
+Person::Person(const std::string &name, int age)
+{
+    age_ = new int(1);
+    *age_ = age;
```

Comment:
You can just combine it as `age_ = new int(age);` to have age_ directly initialize with the value of `age`

Reply:
done